docs/guides/storage.md

```diff
@@ -116,6 +116,9 @@ Note that the original virtual environment is still installed in `$SCRATCH/sqfs-
 
 A benefit of this approach is that the squashfs file can be copied to a location that is not subject to the Scratch cleaning policy.
 
+??? warning
```

Comment:
Starting the admonition with `???` will make the box closed by default.
Use `!!!` to make it open.
Or, if you want it to be closed, you can give it a title like:

`??? warning "virtual environments are normally not relocatable"`